hack/setenv.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+function workflowNamespace {
+  default="sonataflow-infra"
+  read -p "Enter workflow namespace (default: $default): " value
+
+  if [ -z "$value" ]; then
+      workflow_namespace="$default"
+  else
+      workflow_namespace="$value"
+  fi
+  echo "export WORKFLOW_NAMESPACE=$workflow_namespace" > .env
+}
+
+function k8sUrl {
+  url="$(oc whoami --show-server)"
+  echo "export K8S_CLUSTER_URL=$url" >> .env
+}
+
+function k8sToken {
+  sa_namespace="orchestrator"
+  sa_name="orchestrator"
+  read -p "In which namespace we have or have to create the SA holding the persistent token? (default: $sa_namespace): " selected_ns
```

Comment:
worth mentioning that if the NS does not exists, it will be created

Reply:
Is it better?